browser_utils.js

```diff
@@ -537,8 +620,18 @@ async function workaround_setContent(page, html) {
     await waiter;
 }
 
+
 // Render HTML code as a PDF file.
 // modifyPage can be an async function to change the page in the browser.
+
+/**
+ * Render a HTML string as PDF.
```

Comment:
This could make it more clear that the output is a file.

Reply:
Clarified.